src/Humanizer.Tests/DateHumanizeDefaultStrategyTests.cs

```diff
@@ -94,7 +94,7 @@ public void HoursFromNowNotTomorrow(int hours, string expected)
         [Theory]
         [InlineData(1, "yesterday")]
         [InlineData(10, "10 days ago")]
-        [InlineData(28, "28 days ago")]
+        [InlineData(27, "27 days ago")]
```

Comment:
Thanks for the review. Yes, you're right. Suddenly the unit tests started failing!! Do'h!